peps/pep-0774.rst

```diff
@@ -0,0 +1,255 @@
+                    aarch64-unknown-linux-gnu.h
+                    i686-pc-windows-msvc.h 
+                    x86_64-apple-darwin.h
+                    x86_64-pc-windows-msvc.h 
+                    x86_64-pc-linux-gnu.h
+
+Workflow
+--------
+
+To cover the workflow changes, this section can be divided into two parts -
+building CPython with the JIT enabled and working on the JIT’s implementation. 
+
+Building CPython with the JIT
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+Precompiled JIT stencil files will be stored in the ``Tools/jit/stencils``
+directory, with each file name corresponding to its target triple as outlined
+above. At build time, we determine whether to use the checked in stencils or to
+generate a new stencil for the user’s platform. Specifically, for contributors
+with LLVM installed, the ``build.py`` script in ``Tools/jit/stencils`` will allow
+them to regenerate the stencil for their platform. Those without LLVM can rely
+on the precompiled stencil files directly from the repository. 
+
+Working on the JIT’s implementation (or touching JIT files)
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+In CI, stencil files will be automatically validated and updated when changes
+are made to JIT-related files. When a pull request is opened that touches these
+files, the ``jit.yml`` workflow, which builds and tests our builds, will run as
+usual. 
+
+However, as part of this, we will introduce a new step that diffs the current
+stencils in the repo against those generated in CI. If there is a diff for a
+platform’s stencil file, a patch file for the updated stencil is generated and
+the step will fail. Each patch is uploaded to GitHub Actions. After CI is
+finished running across all platforms, the patches are aggregated into a single
+patch file for convenience. You can download this aggregated patch, apply it
+locally, and commit the updated stencils back to your branch. Then, the
+subsequent CI run will pass.
+
+Reference Implementation
+========================
+
+Key parts of the `reference implementation <https://github.com/python/cpython/pull/129331>`__ include: 
+
+- |CI|_: The CI workflow responsible for generating stencil patches.
+
+- |jit_stencils|_: The directory where stencils are stored.
+
+- |targets|_: The code to compile and parse the templates at build time.
+
+.. |CI| replace:: ``.github/workflows/jit.yml``
+.. _CI: https://github.com/python/cpython/blob/main/.github/workflows/jit.yml
+
+.. |jit_stencils| replace:: ``Tools/jit/stencils``
+.. _jit_stencils: https://github.com/python/cpython/blob/main/Tools/jit/stencils
+
+.. |targets| replace:: ``Tools/jit/_targets``
+.. _targets: https://github.com/python/cpython/blob/main/Tools/jit/_targets.py
+
+Ignoring the stencils themselves and any necessary JIT README changes, the
+changes to the source code to support reproducible stencil generation and
+hosting are minimal (around 150 lines of changes).
+
+.. _PEP 774 Rejected Ideas:
+
```

Comment:
Headers are automatically reference targets within the document, so you should be able to simplify this (pt2)
```suggestion
```